README.md

```diff
@@ -117,83 +117,15 @@ iota-sdk = { git = "https://github.com/iotaledger/iota-sdk", branch = "develop"
 
 ## Client Usage
 
-The following example creates a [`Client`](https://docs.rs/iota-sdk/latest/iota_sdk/client/core/struct.Client.html)
-instance connected to
-the [Shimmer Testnet](https://api.testnet.shimmer.network), and retrieves the node's information by
-calling [`Client.get_info()`](https://docs.rs/iota-sdk/latest/iota_sdk/client/core/struct.Client.html#method.get_info),
-and then print the node's information.
-
-```rust
-use iota_sdk::client::{
-    Client,
-};
-
-#[tokio::main]
-async fn main() -> Result<()> {
-    let client = Client::builder()
-        .with_node("https://api.testnet.shimmer.network")? // Insert your node URL here
-        .finish()
-        .await?;
-            
-    let info = client.get_info().await?;
-    println!("Node Info: {info:?}")
-    
-    Ok(())
-}
-```
+The following example creates a Client instance connected to the Shimmer Testnet, and retrieves the node's information by calling `Client.get_info()`, and then print the node's information.
```

Comment:
Feels like this was a mistaken revert

Reply:
No that's fine

Reply:
Also there was no conflict on the README

Reply:
Okie dokie 🤷‍♀️